app/models/s3_csv_export.rb

```diff
@@ -0,0 +1,33 @@
+class S3CsvExport
+  def self.create(rows, filename, headers)
```

Comment:
What do you think about doing `def self.create(rows:, filename:, headers:)`? I know it makes the implementation more verbose, but three's a crowd. Perhaps do this when you add the `acl` in the next PR.